### PR TITLE
FAT2-152 location search

### PR DESCRIPTION
**Pull request by dashton82**

### CODE REVIEW

File: src/SFA.DAS.FAT.Web.UnitTests/Models/WhenCreatingProviderViewModel.cs [hunk 1]
```diff
@@ -17,12 +20,15 @@ public void Then_Maps_The_Fields(Provider source)
...
-                .Excluding(c=>c.NationalOverallCohort));
+                .Excluding(c=>c.NationalOverallCohort)
+                .Excluding(c => c.DeliveryModes));
 
             actual.OverallAchievementRatePercentage.Should().Be($"{(Math.Round(source.OverallAchievementRate.Value)/100):0%}");
             actual.NationalOverallAchievementRatePercentage.Should().Be($"{(Math.Round(source.NationalOverallAchievementRate.Value)/100):0%}");
         }
 
+        // achievement rate
+
```
Comment (PeterDavidWilliams) [inline src/SFA.DAS.FAT.Web.UnitTests/Models/WhenCreatingProviderViewModel.cs:31]: What's this comment for?

File: src/SFA.DAS.FAT.Web.UnitTests/Models/WhenCreatingProviderViewModel.cs [hunk 2]
```diff
@@ -46,5 +52,124 @@ public void Then_Rounds_Values_For_Achievement_Data(Provider source)
             actual.OverallAchievementRatePercentage.Should().Be("39%");
             actual.NationalOverallAchievementRatePercentage.Should().Be("79%");
         }
+
+        // delivery modes
```
Comment (PeterDavidWilliams) [inline src/SFA.DAS.FAT.Web.UnitTests/Models/WhenCreatingProviderViewModel.cs:56]: See above

File: src/SFA.DAS.FAT.Web.UnitTests/Models/WhenCreatingProviderViewModel.cs (hunk 1 above)
Comment (scottywakefield) [reply src/SFA.DAS.FAT.Web.UnitTests/Models/WhenCreatingProviderViewModel.cs:31]: loads of tests in this file, using the comments to organise them a bit

File: src/SFA.DAS.FAT.Web.UnitTests/Models/WhenCreatingProviderViewModel.cs (hunk 2 above)
Comment (scottywakefield) [reply src/SFA.DAS.FAT.Web.UnitTests/Models/WhenCreatingProviderViewModel.cs:56]: see response above

**Review by PeterDavidWilliams — APPROVED** [review]
Approved up to this point. Revisit required once work completed

**Review by PeterDavidWilliams — APPROVED** [review]

**Review by PeterDavidWilliams — APPROVED** [review]

**Pull request merged**